internal/cli/atlas/clusters/watch_test.go

```diff
@@ -31,7 +31,7 @@ func TestWatch_Run(t *testing.T) {
 	mockStore := mocks.NewMockAtlasClusterDescriber(ctrl)
 	defer ctrl.Finish()
 
-	expected := &mongodbatlas.Cluster{StateName: "IDLE"}
+	expected := &mongodbatlas.AdvancedCluster{StateName: "IDLE"}
```

Comment:
we should switch `watch` to use https://docs.atlas.mongodb.com/reference/api/clusters-check-operation-status/
but we can do as separate ticket

Reply:
I am going to open a separate PR for this

Reply:
https://jira.mongodb.org/browse/CLOUDP-93678